### PR TITLE
Family header

### PR DESCRIPTION
**Pull request by ariellam**

### CODE REVIEW

**Review by alexgirard — APPROVED** [review]
lgtm 🥇 💯 🔢 💃 🍔 🔍 :octocat: 🌮 🚦 ⚡️

File: game/family.cc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#include "type.h"
+#include "family.h"
+
+using namespace std;
+
+Family::Family(const Type type): type{type} {}
+
+size_t Family::effectiveness(vector<Family> families) {
+  int strength = 1;
+  for(auto otherFamily : families) {
```
Comment (alexgirard) [inline game/family.cc]: excuse me sir there needs to be a space after the `for` in the loop pls and thank you do you even code 👎

Comment (BenLanglois) [reply game/family.cc]: This is unacceptable
bitmoji

File: game/family.cc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#include "type.h"
+#include "family.h"
+
+using namespace std;
+
+Family::Family(const Type type): type{type} {}
+
+size_t Family::effectiveness(vector<Family> families) {
+  int strength = 1;
+  for (auto otherFamily : families) {
```
Comment (BenLanglois) [inline game/family.cc:12]: Much better
bitmoji

**Review by BenLanglois — APPROVED** [review]

**Pull request merged**